From 8f7373a4a38412611834d30280fcd838bce7cc0c Mon Sep 17 00:00:00 2001
From: Calvin Johnson <calvin.johnson@nxp.com>
Date: Mon, 10 Dec 2018 10:03:51 +0530
Subject: [PATCH 035/235] arm64: dts: ls1012a: reorganize pfe_mac nodes

To keep platform specific properties in the platform dts files,
remove pfe_mac nodes from dtsi and define them in dts files.

Signed-off-by: Calvin Johnson <calvin.johnson@nxp.com>
[Xulin: Original patch taken from NXP LSDK-19.06.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1012a-2g5rdb.dts | 4 ++--
 arch/arm64/boot/dts/freescale/fsl-ls1012a-frdm.dts   | 4 ++--
 arch/arm64/boot/dts/freescale/fsl-ls1012a-frwy.dts   | 4 ++--
 arch/arm64/boot/dts/freescale/fsl-ls1012a-qds.dts    | 4 ++--
 arch/arm64/boot/dts/freescale/fsl-ls1012a-rdb.dts    | 4 ++--
 arch/arm64/boot/dts/freescale/fsl-ls1012a.dtsi       | 5 -----
 6 files changed, 10 insertions(+), 15 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1012a-2g5rdb.dts b/arch/arm64/boot/dts/freescale/fsl-ls1012a-2g5rdb.dts
index 49713b447994..6be3da0eab0d 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1012a-2g5rdb.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1012a-2g5rdb.dts
@@ -89,7 +89,7 @@
 	#address-cells = <1>;
 	#size-cells = <0>;
 
-	ethernet@0 {
+	pfe_mac0: ethernet@0 {
 		compatible = "fsl,pfe-gemac-port";
 		#address-cells = <1>;
 		#size-cells = <0>;
@@ -99,7 +99,7 @@
 		phy-handle = <&sgmii_phy1>;
 	};
 
-	ethernet@1 {
+	pfe_mac1: ethernet@1 {
 		compatible = "fsl,pfe-gemac-port";
 		#address-cells = <1>;
 		#size-cells = <0>;
diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1012a-frdm.dts b/arch/arm64/boot/dts/freescale/fsl-ls1012a-frdm.dts
index b6cdac49ffec..fcc3c271a4d2 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1012a-frdm.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1012a-frdm.dts
@@ -84,7 +84,7 @@
 	#address-cells = <1>;
 	#size-cells = <0>;
 
-	ethernet@0 {
+	pfe_mac0: ethernet@0 {
 		compatible = "fsl,pfe-gemac-port";
 		#address-cells = <1>;
 		#size-cells = <0>;
@@ -94,7 +94,7 @@
 		phy-handle = <&sgmii_phy1>;
 	};
 
-	ethernet@1 {
+	pfe_mac1: ethernet@1 {
 		compatible = "fsl,pfe-gemac-port";
 		#address-cells = <1>;
 		#size-cells = <0>;
diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1012a-frwy.dts b/arch/arm64/boot/dts/freescale/fsl-ls1012a-frwy.dts
index c6201bab8939..712cf5414192 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1012a-frwy.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1012a-frwy.dts
@@ -139,7 +139,7 @@
 	#address-cells = <1>;
 	#size-cells = <0>;
 
-	ethernet@0 {
+	pfe_mac0: ethernet@0 {
 		compatible = "fsl,pfe-gemac-port";
 		#address-cells = <1>;
 		#size-cells = <0>;
@@ -150,7 +150,7 @@
 		phy-handle = <&sgmii_phy1>;
 	};
 
-	ethernet@1 {
+	pfe_mac1: ethernet@1 {
 		compatible = "fsl,pfe-gemac-port";
 		#address-cells = <1>;
 		#size-cells = <0>;
diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1012a-qds.dts b/arch/arm64/boot/dts/freescale/fsl-ls1012a-qds.dts
index 382df05ff813..a4fed83900ea 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1012a-qds.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1012a-qds.dts
@@ -142,7 +142,7 @@
 	#address-cells = <1>;
 	#size-cells = <0>;
 
-	ethernet@0 {
+	pfe_mac0: ethernet@0 {
 		compatible = "fsl,pfe-gemac-port";
 		#address-cells = <1>;
 		#size-cells = <0>;
@@ -152,7 +152,7 @@
 		phy-handle = <&sgmii_phy1>;
 	};
 
-	ethernet@1 {
+	pfe_mac1: ethernet@1 {
 		compatible = "fsl,pfe-gemac-port";
 		#address-cells = <1>;
 		#size-cells = <0>;
diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1012a-rdb.dts b/arch/arm64/boot/dts/freescale/fsl-ls1012a-rdb.dts
index d7f13bc976c7..f112ea68f2be 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1012a-rdb.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1012a-rdb.dts
@@ -53,7 +53,7 @@
 	#address-cells = <1>;
 	#size-cells = <0>;
 
-	ethernet@0 {
+	pfe_mac0: ethernet@0 {
 		compatible = "fsl,pfe-gemac-port";
 		#address-cells = <1>;
 		#size-cells = <0>;
@@ -63,7 +63,7 @@
 		phy-handle = <&sgmii_phy>;
 	};
 
-	ethernet@1 {
+	pfe_mac1: ethernet@1 {
 		compatible = "fsl,pfe-gemac-port";
 		#address-cells = <1>;
 		#size-cells = <0>;
diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1012a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls1012a.dtsi
index d0d22b6030cd..2196c2959036 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1012a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1012a.dtsi
@@ -529,11 +529,6 @@
 		clock-names = "pfe";
 
 		status = "okay";
-		pfe_mac0: ethernet@0 {
-		};
-
-		pfe_mac1: ethernet@1 {
-		};
 	};
 
 	firmware {
-- 
2.17.1

